torch/_torch_docs.py

```diff
@@ -4633,7 +4633,7 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.randint,
            r"""
-randint(low=0, high, size, \*, generator=None, out=None, \
+randint(low=0, high, size, generator=None, out=None,
```

Comment:
the * signals keyword-only arguments. i think we should figure out how to display * rather than kill it.

Reply:
Is it necessary to indicate that generator, etc. are keyword-only arguments? I thought perhaps the inclusion of default values is enough of an indicator.

Reply:
no, having default values is different than being keyword-only arguments. 
```py
def f(a, b=3, *, c=4):
  pass
```
You can call `f(1, 1)` but not `f(1,1,1)`

Reply:
I see, thanks for pointing that out. How about putting it in double-back-quoates? Like
```
randint(low=0, high, size, ``*``, generator=None, out=None,
```

Reply:
Does it render properly? Could you build the doc and share a screenshot?